### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Vayts**
https://vayts.github.io/layout_landing-page/

### CODE REVIEW

**Review by tiserett — CHANGES_REQUESTED** [review]
Hey, looks like a good work to me, but I managed to find few problems ;)

- Burger icon size
- 1260px, you use this value a lot, create variable for it
- Read about using mixins for writing better media queries 
- Even on smaller screens you have hover effects (how would you hover on the phone😀?) Lets leave them only for laptop + bigger screen
- Images looks bad on smaller screens  
- When i move mouse away from hovered element, I should see animation (now I see animation only on hover)

Don't forget to fix other comments + look through your styles, you might want to use some other mixins for font settings etc.

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Add demo link to pull request description

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Good job!
Some comments:
- Logo in header redirects me to 404 error page
- Logo in the opened menu should also be the link
- Check correct styles for these links in the opened menu according to figma (https://prnt.sc/iTmE4JlJNhfM)
- Each link in the opened menu should redirect me to some section. for now even mobile menu is not closing when I click on the link
- Let's limit the size for the images on the medium screens (https://prnt.sc/UKVkVBuX1Aqy)

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
- add smooth scrolling to your page
- will be better to add some smooth hover effects for all links near the contact form
- check correct font styles according to the figma (https://prnt.sc/NhdFnuFWjBpU)
- will be better to limit image size on the medium screens (https://prnt.sc/yYZLzC-kszI7)

**Comment by Vayts** [comment]
hi! thank u for your review! what's wrong with font styles?=(

File: src/styles/_header.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  &__button {
+    font-size: 18px;
+    text-decoration: none;
+    color: inherit;
+
+    @include for-tablet {
+      &:hover {
+        transform: scale(1.1);
+        transition: all 0.1s;
```
Comment (dffUqp) [inline src/styles/_header.scss:68]: better to move this transition outside `:hover`

**Review by dffUqp — APPROVED** [review]
Something wrong with these images but you did really good work so I will approve it👍

Figma:
image

Current page:
image

**Pull request closed without merging**